### DISCUSSION TITLE
Create Game Won't add Snakes

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#219] [Bug Reports]
The Create Game seems to be broken right now. Clicking the `+` to add snakes doesn't seem to be working for me

I see two jQuery errors in my console about `tooltips` and `autocomplete` that I'm not sure if they are related!

**Comment by andrewdmaclean** [comment]
Looking into this now @coreyja

**Comment by andrewdmaclean** [comment]
@coreyja looks like it is fixed now :)
